src/main/java/build/buildfarm/instance/stub/StubInstance.java

```diff
@@ -309,6 +313,11 @@ public void stop() throws InterruptedException {
     if (retryService != null && !shutdownAndAwaitTermination(retryService, 10, TimeUnit.SECONDS)) {
       logger.log(Level.SEVERE, format("Could not shut down retry service for %s", identifier));
     }
+    byteStreamExecutorService.shutdown();
```

Comment:
Prefer the use of `shutdownAndAwaitTermination` as above

Reply:
Done.